src/ajitSJune20/TestPalindromeNumber.java

```diff
@@ -0,0 +1,33 @@
+package ajitSJune20;
+
+public class TestPalindromeNumber {
+
+	static boolean isPalindromeInteger(int number) {
```

Comment:
in my opinion .. u are checking reverse number with provided number two times .. we can set both the things reverse+palindrome in main also..

Reply:
yes we can write every thing in main but for standard practice and guideline of Kirishna, we prefer to create method to perform operation/action.

Reply:
Okay np.. bro.. its good u used the same function for both..

Reply:
Code Reusability looks good to me @AjitSJune20